ph5/utilities/tests/test_segd2ph5.py

```diff
@@ -6,24 +6,28 @@
 import sys
 import shutil
 import tempfile
+import logging
+from StringIO import StringIO as StringBuffer
 from mock import patch
 from ph5.utilities import segd2ph5, tabletokef
 from ph5.core import experiment, segdreader
-from ph5.core.tests.test_base import LogTestCase
+from ph5.core.tests.test_base import LogTestCase, import TempDirTestCase
 
 
-class TestSegDtoPH5(LogTestCase):
+class TestSegDtoPH5(TempDirTestCase, LogTestCase):
     def initialize_ph5(self, editmode):
         EX = experiment.ExperimentGroup(nickname="master.ph5")
         EX.ph5open(editmode)
         EX.initgroup()
         return EX
 
     def setUp(self):
-        # create tmpdir
-        self.home = os.getcwd()
-        self.tmpdir = tempfile.mkdtemp() + "/"
-        os.chdir(self.tmpdir)
+        super(TestSegDtoPH5, self).setUp()
```

Comment:
`super()` call with no args.

Reply:
why? It works this way. If I remove the arguments from super(), it gives me the error:
TypeError: super() takes at least 1 argument (0 given)

Reply:
Sorry, you are correct for python 2.7